scripts/ensmallen-release.sh

```diff
@@ -131,6 +131,15 @@ git add CONTRIBUTING.md;
 git add HISTORY.md;
 git commit -m "Update and release version $MAJOR.$MINOR.$PATCH.";
 
+changelog_str=`cat HISTORY.md |\
```

Comment:
Looks like an easy one.

Reply:
I dunno, maybe it could be improved. :smile:  When I looked into what it took to write a new custom Github action, I decided it would be easier just to use tools I already know... maybe we can come back and make it better later...

Reply:
Not sure, I thought GitHub action is an interface to spawn a container to run the code, basically Jenkins. Anyway, what I meant to say was that the command pipeline to parse the History file looks "easy" not the build pipeline :)

Reply:
Yeah, that's kind of what I thought---I could write bash scripts that run in a container that get fired off by some action... or I could use mlpack-bot to write bash/JS scripts that get fired off by some action.  Since I already have mlpack-bot set up and watching the right events, this seemed like the easier way to go. :)  I'll commit the mlpack-bot code soon---I now have it drafting releases correctly.  The last step is to fire off the script that updates the website, and then I think this is ready to actually try out. :)